components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx

```diff
@@ -102,9 +102,8 @@ export default class CustomTimeRangeSelector extends React.Component {
 
     handleStartTimeChange(date) {
         this.startTime = date;
-        const start = Moment(this.startTime);
-        const end = Moment(this.endTime);
-        if(end.diff(start) < 0) {
+        if(Moment(this.startTime,'YYYY-MM-DD HH:mm:ss.000').unix() >=
```

Comment:
Format the code segments. Need a space after if.

Reply:
This code has been refactored with PR #270